back/src/Guide/Logger/Init.hs

```diff
@@ -0,0 +1,68 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE StandaloneDeriving  #-}
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE FlexibleContexts    #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
When there are orphans, let's also always add a comment like this so that it would be easier to check if the pragma is outdated or not:

```haskell
{-# OPTIONS_GHC -fno-warn-orphans #-} -- instance Read Df1.Level
```